homeassistant/components/media_player/kodi.py

```diff
@@ -61,6 +61,7 @@
 # https://github.com/xbmc/xbmc/blob/master/xbmc/media/MediaType.h
 MEDIA_TYPES = {
     'music': MEDIA_TYPE_MUSIC,
+    'audio': MEDIA_TYPE_MUSIC,
```

Comment:
Let's put this at the bottom with a comment similar to `channel` explaining why it's included in this list, since it's not in the linked source list.